Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/spark/stage/Stage.java

```diff
@@ -257,4 +261,6 @@ public long getInputRecords() {
     public void setInputRecords(long inputRecords) {
         this.inputRecords = inputRecords;
     }
+
+    public static final List<Stage> EMPTY_LIST = Collections.unmodifiableList(new ArrayList<>(0));
```

Comment:
>EMPTY_LIST [](start = 36, length = 10)

Why we need this? Can we put it into some helper function instead of adding one for each class?